### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by VigneshSrinivasan10** [team-approx-bayes/dl-with-bayes#1]
Dear Authors, 

Thank you very much for the amazing work and the code! 
It would be great if you could upload the pretrained models for `Image classification` tasks.

Thank you in advance,

**Comment by kazukiosawa** [comment]
@VigneshSrinivasan10 
Thank you for your suggestion! 
We will upload the pertained models. Please be patient as it takes time to prepare.

**Comment by VigneshSrinivasan10** [comment]
@kazukiosawa 
Thank you very much for your reply. 

In the meantime, I trained a model on CIFAR10 dataset as described in your repository. These are the results after training the model with `configs/cifar10/lenet_vogn.json` setting for `100` epochs. And these are the results I get: 

`Eval: Average loss: 1.8339, Accuracy: 6904/10000 (69.04%)`
 
I find that the config file: `configs/cifar10/lenet_vogn.json` corresponds to `VOGN+BN`. However, I would like to reproduce the results from the paper of the setting `VOGN+momentum+BN`, where you get the best results. 

It would be great if you update the config file to be compatible with that setting or point me in the direction of how to do it. 

Thanks!

**Comment by jmamath** [comment]
Dear authors,

I would like to know if it is still possible to get the pretrained weights ?
